transmart-server/grails-app/conf/application.groovy

```diff
@@ -323,6 +323,36 @@ grails { plugin { springsecurity {
 } } }
 /* }}} */
 
+/* {{{ GlowingBear query subscription configuration */
+grails {
+    mail {
+        host = "smtp.gmail.com"
+        'default' {
+            from = "subscription.gb@gmail.com"
+        }
+        port = 465
+        username = "subscription.gb@gmail.com"
+        password = "gb@TheHyve"
```

Comment:
Please change credentials and remove this line here. You don't want people spamming from this account.